containers/Ideas/Idea.js

```diff
@@ -28,7 +28,10 @@ export const Idea = (props) => {
     {
       title: 'Edit',
       icon: 'pencil',
-      action: () =>  props.navigation.navigate('IdeaCreate', {idea: idea, project: project, editing: true}),
+      action: () =>  {
+        toggleMenu();
```

Comment:
Good catch! But do we really want `toggleMenu` or do we want `setMenuVisible(false)` like when we open the deleteModal? The latter seems more robust to me.

Reply:
I guess both work, but we should probably decide on one way. Would you then also use `setMenuVisible(false)` when clicking the Cancel button @fuzzylogic2000 ?

Reply:
Oh right, yes, I think! I would only use the toggle when there is a button that opens and closes something. If we have sth. that only closes it, I would go for `setMenuVisible(false) `, because that's what we want there. @Rineee

Reply:
yes, totally agree!